client/my-sites/importer/site-importer/site-importer-input-pane.jsx

```diff
@@ -91,60 +71,6 @@ class SiteImporterInputPane extends React.Component {
 		this.validateSite();
 	}
 
-	// TODO This can be improved if we move to Redux.
-	UNSAFE_componentWillReceiveProps = nextProps => {
```

Comment:
The `UNSAFE_componentWillMount` code can be safely moved to `componentDidMount`. (GitHub won't let me comment on that line, so I'm commening on the closes commentable line 🙂 )